components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -150,7 +165,7 @@ class AdminSidebar extends React.PureComponent {
 
     visibleSections = () => {
         const {config, license, buildEnterpriseReady, consoleAccess, adminDefinition, cloud} = this.props;
-        const isVisible = (item) => {
+        const isVisible = (item: any) => {
```

Comment:
Unfortunately I couldnt find the type. When `console.log`, I see different types such as `string`, `function` or `object` so I kept as `any`. Please let me know if you have any suggestion.

Reply:
You could do something with the `AdminDefinition`: `AdminDefinition[keyof AdminDefinition]`
(you might need to make a type alias for AdminDefinition)

Reply:
I tried to do `typeof AdminDefinition[keyof typeof AdminDefinition]` but it didn't work. As you suggested best way to solve this issue is to create a type alias for AdminDefinition. Can you show me a good example of types created in Mattermost. Maybe [Plugins?](https://github.com/mattermost/mattermost-webapp/blob/master/types/store/plugins.ts)

Reply:
As suggested by @hmhealey I will keep this as `any`. After the migration of `admin_definition.jsx` internally this could be updated